sdk/nodejs/types/input.ts

```diff
@@ -21839,12 +21537,6 @@ export namespace storage {
        */
       metadata?: pulumi.Input<meta.v1.ObjectMeta>
 
-      /**
-       * Status of the VolumeAttachment request. Populated by the entity completing the attach or
-       * detach operation, i.e. the external-attacher.
-       */
-      status?: pulumi.Input<storage.v1beta1.VolumeAttachmentStatus>
```

Comment:
Will this be a breaking change for anybody who was setting `.status` for some reason?

Reply:
Why would it? You can't set `.status`.